### DISCUSSION TITLE
Authenticating fonoster on my provider for incoming calls

### DISCUSSION THREAD

**Discussion by yuvalkry** [fonoster/yaps#596] [Q&A]
Hello,

 What is the way to create the link between my virtual number and api.fonoster.io?
 There is something I have to set up so it will communicate with Fonoster when a call is coming.

 What do I have to set up on the provider so it will communicate with Fonoster?
 Is api.fonoster.io a SIP server?
 What credentials to use?

 I did not find that information in the documentation.

   Thanks, 
         Yuval

**Comment by psanders** [comment]
In your provider side you want to find an set the Static IP Authentication using Fonoster's IP. With VoIP.ms, for example, it looks like this

<img src="https://github.com/fonoster/fonoster/assets/539774/51a72963-2750-49dc-a4eb-b7c39e410a1b" alt="image" width="50%" height="auto">

In addition to that you need to set the Dialing Mode to `E164`

**Comment by yuvalkry** [comment]
Thank you!

I am afraid there is something basic that I do not understand.

voip.ms documentation:
   https://blog.voip.ms/blog/what-is-ip-authentication-and-how-it-works/
says:

Once you select “Static IP” for the option “Authentication type” you will be able to enter the WAN IP address of the network your calls will be made from. With this done, the Voip.ms servers will not allow invites from this username if they are not coming from the specified IP address.

So this setting is about calls that go fonoster->voip.ms  .

When I dial a number, it first reaches my provider (if I understand correctly).
Then, for example voip.ms will forward to fonoster - so this is the reverse direction.
Is the setting of static IP relevant also when dialing the number?

What I would like to do is to dial a number and get the call forwarded to the webhook.
I did not see a tutorial of how to configure it. It would be very helpful if you point to information.

Thanks,
Yuval
